src/components/DocLink/DocLink.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react"
+import { VStack } from "@chakra-ui/react"
+import { Meta, StoryObj } from "@storybook/react"
+
+import DocLink from "."
+
+const meta = {
+  title: "Atoms / DocLink",
```

Comment:
I'm not sure we want this at the Atoms level:

<img width="274" alt="image" src="https://github.com/user-attachments/assets/1483d276-445f-40b7-b166-6bd2eb837669" />

https://github.com/alexandriaroberts/ethereum-org-website/blob/a643d2f21550959649d6776174345edc4afcaea7/src/components/ui/__stories__/Link.stories.tsx#L8

Makes sense to me to put this one under navigation molecule instead:

```suggestion
  title: "Molecules / Navigation / DocLink",
```

<img width="333" alt="image" src="https://github.com/user-attachments/assets/85934b2c-a6ba-4c96-a787-d4f938346a54" />